packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -43,6 +66,7 @@ class SyftWorker(SyftObject):
     image_hash: Optional[str]
     healthcheck: Optional[WorkerHealth]
     status: WorkerStatus
+    image: SyftWorkerImage
```

Comment:
Similarly, we no longer need `image_hash: Optional[str]` here since `image: SyftWorkerImage` already contains it.

Reply:
That's true, thanks @kiendang . Updated the code